content/en/notebooks/notebooks_new.md

```diff
@@ -0,0 +1,351 @@
+
+## Multiplayer experience in Notebooks
+
+Notebooks are collaborative, allowing multiple users to edit simultaneously. If a collaborator opens your document, their cursor will appear inside the document. Hover over a cursor to see the collaborator's name.
+
+<!-- TODO Add updated image from new notebooks -->
+
+### Widgets
+
+When another user is editing a widget, an outline appears to indicate this. Since widgets are saved as last write wins, avoid editing a widget that someone else is working on.
+
+<!-- TODO Add updated image from new notebooks -->
+
+#### Presence
+
+At the top of the document you can see a a row avatar images of all the users currently viewing the document. Hover over an avatar to see the name of the associated collaborator.
+
+<!-- TODO Add updated image from new notebooks -->
+
+## Configuring a notebook
+
+### Template variables
+
+Notebooks support template variables. Dynamically scope visualizations by adding and selecting template variable values. For more information, see [Template Variables][17].
+
+### Time controls
+
+By default, all graph cells are linked to the global time frame set in the notebook header.
+
+To view a different time frame, select an option in the global time picker, or scrub on a graph directly. The notebook URL will update to reflect this new time frame without saving it to the notebook.
+
+**Note**: Clicking and dragging to zoom in on a graph does not unlock the cell from the global time. It changes the notebook's global time instead.
+
+<!-- TODO Add updated image from new notebooks -->
+
+To save this time as the notebook's default, click **Set Default Time**. To reset your global time back to the previously saved default global time, click the reset button.
+
+Individual cells can be unlinked from the global time and set to an independent time frame.
+
+<!-- TODO Add updated image from new notebooks -->
+
+To view a different time frame on a single cell, edit the cell and use the toggle to unlink it from Global Time. Change the time frame using the time picker or by scrubbing on the graph. Changes made in edit mode are saved automatically when you click **Done**. To discard your changes, click **Cancel** instead of **Done**.
+
+### Modes
+
+You can switch between modes from within the notebook by selecting the dropdown at the top right of your notebook.
+
+- **Editing**: Make changes to the notebook.
+- **Viewing**: Contents are Read Only, preventing users from making unwanted edits to existing configurations and information.
+
+### Version history
+
+From a notebook, click the **Configure** icon and click **Version history** to open the Version History side panel. You can preview, restore, or clone your notebook's version history. For more information, see the [Version History guide][18].
```

Comment:
Is the **Configure** icon different from the cog icon? If they're the same thing, I like how you referred to it here - I like referring to what icons are for, so people can think about the objectives they're trying to accomplish, instead of looking for specific icons that can be harder to remember if there are many of them.